update_rate: 10
diag_config:
  name: "internet_connection"
  hardware_id: "wifi_router"
  msg_ok: ""
  msg_warn: "connection is unstable"
  msg_error: "connection is lost"
  msg_stale: ""
